dbt_platform_helper/commands/pipeline.py

```diff
@@ -178,29 +173,6 @@ def _generate_codebase_pipeline(
     )
 
 
-def _generate_copilot_environments_pipeline(
-    app_name, codestar_connection_arn, git_repo, configuration, base_path, pipelines_dir, templates
-):
-    makedirs(pipelines_dir / "environments/overrides", exist_ok=True)
-
-    template_data = {
-        "app_name": app_name,
-        "git_repo": git_repo,
-        "codestar_connection_arn": codestar_connection_arn,
-        "pipeline_environments": configuration,
-    }
-
-    _create_file_from_template(
-        base_path, "environments/buildspec.yml", pipelines_dir, template_data, templates
```

Comment:
I think all of these copilot env pipeline template files can also be removed
https://github.com/uktrade/platform-tools/tree/DBTP-1290/dbt_platform_helper/templates/pipelines/environments